app/views/alaveteli_pro/info_request_batches/_progress_bar.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="batch-summary">
+  <% phases.each do |label, count| %>
+    <% if count > 0 %>
+      <div class="batch-summary__phase
+                  batch-summary__phase--<%= label.parameterize %>"
+           style="flex-grow: <%= count %>">
```

Comment:
I wonder whether we need the vendor-prefixed versions here too? Quick google suggests `-webkit-flex-grow` for Safari (desktop and mobile) 2014 and earlier, and `-ms-flex-positive` (!!) for IE10.